NitroxClient/GameLogic/EquipmentSlots.cs

```diff
@@ -24,18 +26,40 @@ public void BroadcastEquip(Pickupable pickupable, GameObject owner, string slot)
             string itemGuid = GuidHelper.GetGuid(pickupable.gameObject);
             byte[] bytes = SerializationHelper.GetBytes(pickupable.gameObject);
 
-            EquippedItemData equippedItem = new EquippedItemData(ownerGuid, itemGuid, bytes,slot);
-            EquipmentAddItem equipPacket = new EquipmentAddItem(equippedItem);
-            packetSender.Send(equipPacket);
+            EquippedItemData equippedItem = new EquippedItemData(ownerGuid, itemGuid, bytes, slot);
+
+
+            Player player = owner.GetComponent<Player>();
+            if (player != null)
+            {
+                EquipmentAddItem equipPacket = new EquipmentAddItem(equippedItem,true);
```

Comment:
Can we simplify this to:


```
bool isPlayerEquipment = (player != null);
EquipmentAddItem equipPacket = new EquipmentAddItem(equippedItem, isPlayerEquipment);
packetSender.Send(equipPacket);
```


That would allow us to get rid of the redundant code.